src/pages/workspace/tags/WorkspaceTagsPage.tsx

```diff
@@ -107,7 +107,7 @@ function WorkspaceTagsPage({policyTags, route}: WorkspaceTagsPageProps) {
                         title={translate('workspace.common.tags')}
                         shouldShowBackButton={isSmallScreenWidth}
                     />
-                    <View style={[styles.ph5, styles.pb5]}>
+                    <View style={[styles.ph5, styles.pb5, styles.pt3]}>
```

Comment:
Same here, is this intended?

Reply:
yes. It was pointed out to me that my layout was slightly off compared to designs and other pages. Turns out other pages were also slightly off and the description below title was missing 12px padding-top